.github/labeler.yml

```diff
@@ -118,6 +185,25 @@
         - pkgs/top-level/haskell-packages.nix
         - pkgs/top-level/release-haskell.nix
 
+"6.topic: java":
+  - any:
+    - changed-files:
+      - any-glob-to-any-file:
+        - nixos/modules/programs/java.nix
+        - pkgs/development/compilers/openjdk/**/*
```

Comment:
```suggestion
        - pkgs/development/compilers/graalvm/**/*
        - pkgs/development/compilers/openjdk/**/*
```

Reply:
also `pkgs/development/compilers/zulu` and `pkgs/development/compilers/temurin-bin`